Swiift-project1/Services/Service.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  Service.swift
+//  Swiift-project1
+//
+//  Created by Kezikov Boris  on 07.03.2020.
+//  Copyright © 2020 epam. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+
+
+class Service {
+    
+    public static func generateRandomValue(leftBorder: Int, rightBorder:Int) -> Int {
+        let tmp = Int.random(in: leftBorder..<rightBorder)
+        print(tmp)
+        return tmp
+    }
+    public static func validateInput(value: String) -> Bool {
+        if (value.elementsEqual("")){
```

Comment:
use the optional type in arguments 
checking for an empty line here and passing an empty line from `onClick` - a crutch

```
guard let value = value else {
    return
}
```